index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="pl">
+<head>
+    <meta charset="utf-8" />
+    <title>CHARACTERS-CREATOR</title> 
+    <meta name="description" content="Na tej stronie stworzysz swojego pierwszego avatara!!" />
+    <meta name="keywords" content="avatar, postać, kreator"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"  />
+    <link rel="stylesheet" href="style.css" type="text/css" />   
+</head>
+<body>
+    <form>
+        <h1>Stwórz swoją własną postać!!!</h1>
+        <label class="name-style" for="name">Imię:</label><br>
+        <input type="text" class="your-name" name="y-name" value=""><br>
```

Comment:
very good, however we have a small issue here.
`for` attribute in `label` is pointing to `id` of element described. So, `input` needs `id="name"` for this case. Analogical to another inputs.